roles/restore/tasks/deploy_galaxy.yml

```diff
@@ -14,66 +14,27 @@
 
 - name: Set custom resource spec variable from backup
   set_fact:
-    cr_spec: "{{ cr_object.stdout }}"
+    cr_spec_from_backup: "{{ cr_object.stdout }}"
     cr_spec_strip: "{ "
```

Comment:
We should remove this `cr_spec_strip` var, it looks like it is no longer used.

Reply:
Same with the other variables here that seem to have been for building the cr_spec... unless I've misunderstood this.